src/components/Spinner/README.md

```diff
@@ -135,6 +135,8 @@ function SpinnerWithFocusManagement() {
 
 SVGs are often conveyed inconsistently to assistive technologies. The `Spinner` component's accessibility is also highly contextual. When the parent component is focusable, you'll need to set the `hasFocusableParent` prop for the appropriate `role` attribute to be applied.
 
+For optimal user experience, make use of the `accessibilityLabel` prop. It’ll allow assisted technology to provide context.
```

Comment:
```suggestion
For optimal user experience, make use of the `accessibilityLabel` prop to let assistive technology users know the purpose of the spinner.
```

A tweak to provide some context. @sadiesaurus, any thoughts?

Reply:
This is great! Could also replace "make use of" with "use" to keep it simple and direct.